install.md

```diff
@@ -5,7 +5,7 @@ However, if you'd like to try building it from source, follow the steps below.
 
 ## Fedora, CentOS, RHEL, and related distributions:
 
-```bash
+``` bash
```

Comment:
I don't hink we need those additional spaces, right?

Reply:
rebase done